### PR TITLE
Page Content 2 Column Link Card

### LINKED ISSUE #44: "Page Content" 2 Column Link Card - note the optional list of links under the card OR the single external link - a card can be of either type

### PR DESCRIPTION
**Pull request by joekrill**
# Adds 2 Column Link Card

Screen Shot 2022-05-26 at 1 50 38 PM

Mobile:

Note: I moved the tags below the title based on the discussion from yesterday.

---

Closes #44

### CODE REVIEW

**Review by alse — CHANGES_REQUESTED** [review]
@joekrill , I just left some small suggestions. Looks like my tailwind container updates didn't make it in the last one, it should fix some responsive issues. I opened a new pr with it: #59

File: libs/design-system/src/lib/Components/LinkCard2Column/index.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+              <Tag key={tag} name={tag} />
+            ))}
+          </div>
+        )}
+        <p>{description}</p>
+        <ButtonLink href={buttonUrl} className="mb-10 mt-2 px-8 py-3 md:mt-10">
+          {buttonText}
+        </ButtonLink>
+      </div>
+      <div className="flex flex-1 flex-col items-stretch">
```
[the remark this answers is not in the document]
Comment (joekrill) [reply libs/design-system/src/lib/Components/LinkCard2Column/index.tsx:41]: Not sure I totally understand.  These should be full width in those cases already, I believe.  Do you mean the outer card should continue to stack vertically at `md` and not switch to horizontal until `lg` (i.e. replace `md:flex-row` with `lg:flex-row` on the container)?

**Comment by joekrill** [comment]
I think I've got all the changes covered. I had to rebase and force-push so hopefully that didn't cause any problems. 

It's not clear from the designs what the hover styles are supposed to be (are they specified somewhere in particular that maybe I'm missing?). In dark mode, in particular, the drop shadow on hover of the "items" is hardly noticeable so there isn't much of a hover effect (I'm hovering the top Item in this shot):

 
Screen Shot 2022-05-31 at 10 14 57 AM


While in light mode it's much more clear:

Screen Shot 2022-05-31 at 10 17 24 AM

Should there be an adjustment made for dark mode hover?

**Review by 10thfloor — APPROVED** [review]

File: libs/design-system/src/lib/Components/LinkCard2Column/LinkCard2Column.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+export const Default = Template.bind({});
+Default.args = {
+  buttonText: 'Get Started',
+  buttonUrl: '#changeme',
+  description:
+    'Building on Flow is easy. Start building now with lorem ipsum et sigitus loranum prospitarius.',
+  title: 'Start Your Project',
+  tags: ['Tag', 'Lorem', 'Ipsum'],
+  items: [
+    {
```
Comment (10thfloor) [inline libs/design-system/src/lib/Components/LinkCard2Column/LinkCard2Column.stories.tsx:30]: Same comment about interfaces per the "3 column PR"

File: libs/design-system/src/lib/Components/LinkCard2Column/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (alse) [reply libs/design-system/src/lib/Components/LinkCard2Column/index.tsx:47]: Ah it only showed up with the container changes from #59 . Not a big deal though, can merge this in

**Review by alse — APPROVED** [review]

**Pull request merged**